validator/validator-main.protoascii

```diff
@@ -3509,7 +3511,7 @@ tags: {
   attr_lists: "input-name-attr"
   # Note that this amp-bind attribute is not available for the AMP4Email format
   # thus we add this manually for the formats that should.
-  attrs: { name: "[type]" }
+  attrs: {name: "[type]"}
```

Comment:
please undo this change as well

Reply:
Same here. not sure what happened here. reverted. perhaps my editor formatting style. reverted.